ngtools/scene.py

```diff
@@ -581,8 +591,20 @@ def load(
                     raise ValueError(
                         "Cannot load local files without a fileserver"
                     )
-                short_uri = fileserver + "/local/" + op.abspath(short_uri)
+                if parsed.format == "trk":
+                    short_uri = fileserver + "/trk/local/" + op.abspath(
```

Comment:
Have you tried this case? I am thinking it should be `/trk/file/` instead of `/trk/local/` (unless fsspec understands `local` as an alias for `file`)